.github/workflows/ci.yml

```diff
@@ -8,28 +8,31 @@ name: Build and test
 # time we only run the full system tests on Ubuntu (although this can easily be
 # changed).
 
-
 on:
   push:
     branches: ["main"]
   pull_request:
   merge_group:
+  schedule:
+    - cron: "0 9 15 * *"
```

Comment:
Since the build is brittle, I propose we run it monthly to keep vaguelly on top of things...?